CHANGES_NEXT_RELEASE

```diff
@@ -17,6 +17,7 @@
 - Add: Support for null values in string filters (#2359)
 - Fix: Partial sanity check for custon url, only if the url contains no replacements using ${xxx} (#2280, #2279)
 - Fix: if entity::type is given but as an empty string, in payload of POST /v2/subscriptions or PATCH /v2/subscriptions/{sub}, an error is returned (#1985)
+- Fix: more intelligent, quote-aware, operator lookup inside q/mq string filters (#2452)
```

Comment:
What about the actual fix? I mean 

"Fix: attribute names including ':' now are supported with the single-quote syntax, e.g. q='devices:location' (#2452)"